charts/kashti/values.yaml

```diff
@@ -18,6 +18,8 @@ service:
   internalPort: 80
 ingress:
   enabled: false
+  # From Kubernetes 1.18+ this field is supported in case your ingress controller supports it. When set, you do not need to add the ingress class as annotation.
```

Comment:
@czunker since you pointed out that for this to work, the ingress controller in question must also support it, I think that's a detail that is worth mentioning here.

Reply:
Nevermind. It clearly says right here "in case your ingress controller supports it." Not sure how I missed that.